### PR TITLE
Edit documentation for user

### PR DESCRIPTION
**Pull request by felixhalim**

### CODE REVIEW

File: docs/UserGuide.md
Comment (ShawnTanzc) [inline docs/UserGuide.md:15]: General notes: 
+ 'convert /to'
+ 'edit functions' (ForEx Rate, ForEx Abbreviation)

File: docs/UserGuide.md
Comment (ShawnTanzc) [inline docs/UserGuide.md:35]: include foreign currency abbreviation

**Review by ShawnTanzc — APPROVED** [review]
LGTM, have to add new functions once merge

**Review by ivanderjmw — APPROVED** [review]
Looks like a good user guide. Pay attention to typos.

**Pull request merged**
